test/testnullpointer.cpp

```diff
@@ -2092,7 +2092,8 @@ class TestNullPointer : public TestFixture {
               "    if (!y) {}\n"
               "  }\n"
               "}\n");
-        ASSERT_EQUALS(
+        TODO_ASSERT_EQUALS(
```

Comment:
fyi it looks like my prototype change in #3014 fixes this behavior (i.e. the resulting errout.str() is "")